R/geom-smooth.r

```diff
@@ -7,7 +7,7 @@
 #'
 #' Calculation is performed by the (currently undocumented)
 #' `predictdf` generic and its methods.  For most methods the standard
-#' error bounds are computed using the [predict()] method - the
+#' error bounds are computed using the [predict()] method -- the
 #' exceptions are `loess` which uses a t-based approximation, and
```

Comment:
commas between the geom and the descriptor (i.e. "`loess`, which..." and "`glm`, where...") I'm less wed to the one with glm, though.